astropy/units/quantity_helper/function_helpers.py

```diff
@@ -584,9 +585,9 @@ def _check_bins(bins, unit):
         return bins
 
 
+@deprecated_renamed_argument('normed', 'density', '4.0')
```

Comment:
Is this a proper way to deprecate `normed`? Seems like this is some `ufunc` magic, so I am not sure.

Reply:
yes, this should work, but let's see the tests follow the theory, too.

Reply:
however, given that density was already a kwarg, you could simply use it without the alternative part to just remove `normed`.

Reply:
I was trying to find example of something like `@deprecated_renamed_argument('normed', None, '4.0')` but I can't find anything; I don't think that use case is even tested. Also, setting second arg to `None` also means `normed` check is still needed in the function, so more work down the line when it is time to completely remove it. 🤷‍♀

Reply:
Well, well, this broke numpy-dev anyway. So, I'll try your suggestion and see. https://travis-ci.org/astropy/astropy/jobs/561161578

Also, the codecov failure makes no sense to me.